byron/chain/formal-spec/blockchain-spec.tex

```diff
@@ -950,10 +993,12 @@ \subsection{Block header processing}
 
 \subsection{Block body processing}
 
-During processing of the block body, we perform two main functions: verification
-of the body integrity using the proofs contained in the block header, and update
-of the various state components. These rules are given in figure
-\ref{fig:rules:bbody}.
+During processing of the block body, we perform two main functions:
+verification of the body integrity using the proofs contained in the block
```

Comment:
I have no problem with this terminology, but my view is that developers, who are a substantial part of the target audience of these specifications, don't feel comfortable talking about proofs. I certainly wans't comfortable until I got into the Curry-Howard isomorphism. This is the first time it occurs in the Byron-Shelley era specifications. Can we use "values" or "terms" instead?

Reply:
A 'proof' here actually refers to a hash, not a term as proof! We could of course just use 'hash' though...not sure!

Reply:
Hash value?

Reply:
Just renamed `Proof` to `Hash` to avoid confusion. Thanks!